MultiTarget/UseTargetFrameworks.ps1

```diff
@@ -3,28 +3,32 @@
     Changes the target frameworks to build for each package created within the repository.
 .DESCRIPTION
     By default, only the UWP, Windows App SDK, and WASM heads are built to simplify dependencies 
-    needed to build projects within the repository. The CI will enable all targets to build a package
+    needed to build projects within the repository. The CI will enable all include to build a package
```

Comment:
Some global find and replace? same on L11 and 19, should check the text here and elsewhere so that it still makes sense?

Reply:
Good catch! Fixed in 7de12bfc9d0377bd6eb3303a65b34219d881db53